provider/openstack/networking.go

```diff
@@ -49,6 +49,10 @@ type Networking interface {
 	// interfaces on the given list of instances.
 	// Needed for Environ.Networking
 	NetworkInterfaces(ids []instance.Id) ([]corenetwork.InterfaceInfos, error)
+
+	// FindNetworks returns a set of names of internal or external network
```

Comment:
s/a set of names of/a set of

"FindNetworks returns a set of internal or external network names..."